src/studio.css

```
@@ -1,24 +1,1239 @@
+  height: 5px;
+  background: currentColor;
+}
+
+.sound-toggle.is-on .sound-bars i {
+  animation: barBounce var(--beat-duration) ease-in-out infinite alternate;
+}
+
+.sound-toggle.is-on .sound-bars i:nth-child(2) { animation-delay: calc(var(--beat-duration) * .2); }
+.sound-toggle.is-on .sound-bars i:nth-child(3) { animation-delay: calc(var(--beat-duration) * .4); }
+
+@keyframes barBounce {
```

Stylelint (17.14.0)

[error] Expected keyframe name "barBounce" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)